src/index.js

```diff
@@ -1,13 +1,17 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import './index.css';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { composeWithDevTools } from 'redux-devtools-extension';
+import combinedReducers from './reducers/index'
 import App from './App';
 import reportWebVitals from './reportWebVitals';
-
+let store = createStore(combinedReducers,composeWithDevTools)
 ReactDOM.render(
-  <React.StrictMode>
+  <Provider store = {store}>
     <App />
-  </React.StrictMode>,
+  </Provider>,
   document.getElementById('root')
 );
 
```

Comment:
I confirmed with the docs that the Provider tag is the outermost wrapper, so it will wrap the BrowserRouter as well.